guides/common/modules/proc_configuring-host-based-access-control-for-freeipa-users-logging-in-to-project.adoc

```diff
@@ -0,0 +1,114 @@
+[id="configuring-host-based-access-control-for-freeipa-users-logging-in-to-project_{context}"]
+= Configuring host-based access control for {FreeIPA} users logging in to {Project}
+
+You can use host-based access control (HBAC) rules to manage access control within your {FreeIPA} domain.
+In {FreeIPA}, HBAC rules define which users can access which hosts and which services can be used to gain access.
+
+For example, you can configure HBAC on the {FreeIPA} server to limit access to {ProjectServer} only to selected users or user groups.
+By configuring a HBAC rule in the {FreeIPA} domain, you can ensure {Project} does not create database entries for users who should not have access.
+
+.Prerequisites
+* {FreeIPA} user account with privileges to configure HBAC rules
+
+.Procedure
+. On the {FreeIPA} server, configure HBAC control.
+ifndef::orcharhino[]
+For more information, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
+endif::[]
+.. Create a HBAC service for {ProjectServer}.
+.. Create a new HBAC rule to define the required access control.
+Add the following {FreeIPA} entities to the HBAC rule:
+... The HBAC service for {ProjectServer}
+... The {ProjectServer} host
+... The users or user groups to whom you want to grant access
+.. Make sure the default {FreeIPA} `allow_all` rule is disabled.
+ifdef::satellite[]
+For information about how to disable `allow_all` without disrupting other services, see the https://access.redhat.com/solutions/67895[How to configure HBAC rules in IdM] article on the Red{nbsp}Hat Customer Portal.
+endif::[]
+. On your {ProjectServer}, load the host-based access control rules from {FreeIPA}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-pam-service=satellite-prod
+----
+
+.Verification
+* Log in to the {ProjectWebUI} as a user defined in {FreeIPA}.
+** If the user is included in the HBAC rule, {ProjectWebUI} will grant access.
+** If the user is not included in the HBAC rule, {ProjectWebUI} will not grant access.
+
+ifndef::satellite[]
+.Additional resources
+* For more information about the `allow_all` rule and configuring HBAC in {FreeIPA}, see link:https://www.freeipa.org/page/Howto/HBAC_and_allow_all[HBAC and `allow_all`] in {FreeIPA} documentation.
```

Comment:
I checked with Filip and he said that in general https://freeipa.readthedocs.io receives more updates than https://www.freeipa.org/page/Documentation.html 
https://freeipa.readthedocs.io/en/latest/workshop/4-hbac.html
Other than that I haven't found anything useful but I think it's okay to link to RH docs since they are publicly available.

Reply:
I changed the link, thanks!